sourcemash/api/feeds.py

```diff
@@ -24,25 +24,28 @@ def output(self, key, feed):
 class getUnreadCount(fields.Raw):
     def output(self, key, feed):
         total_item_count = Item.query.filter_by(feed_id=feed.id).count() 
-        
-        if not current_user.is_authenticated():
-            return total_item_count 
-        
         read_item_count = UserItem.query.filter_by(user=current_user, feed_id=feed.id, unread=False).count()
-
         return total_item_count - read_item_count
 
 feed_fields = {
     'id': fields.Integer,
     'title': fields.String,
     'url': fields.String,
+    'subscribed': isSubscribed
+}
+
+feed_fields_detailed = {
     'description': fields.String,
     'image_url': fields.String,
-    'last_updated': fields.DateTime,
+    'last_updated': fields.DateTime
+}
```

Comment:
The fields in feed_fields_detailed won't require any more time to compile than the fields in feed_fields (i.e. they're just String and DateTime fields). So I think, to simplify things, we should keep description, image_url, and last_updated in the regular feed fields, and remove feed_fields_detailed.